src/modules/data/blocks/ticket/sagas.js

```diff
@@ -98,6 +98,22 @@ export function* setTicketsInitialState( action ) {
 	yield put( actions.setTicketsProvider( provider || defaultProvider ) );
 }
 
+export function* removeTicketsBlock() {
```

Comment:
Might be misunderstanding this, but it sort of looks like we're resetting the ticket. Are we really removing it?

Reply:
I can rename to `resetTicketsBlock` instead, that'll be less confusing.